packages/lodestar-cli/src/cmds/beacon/fileEnr.ts

```diff
@@ -0,0 +1,40 @@
+import {createKeypairFromPeerId, ENR, ENRKey, ENRValue} from "@chainsafe/discv5";
+import {writeFileSync} from "fs";
+import PeerId from "peer-id";
+import {readFileSync} from "../../util";
+
+export class FileENR extends ENR {
+  private filename: string;
+  private localPeerId: PeerId;
+
+  constructor(filename: string, peerId: PeerId) {
+    super();
+    this.filename = filename;
+    this.localPeerId = peerId;
+  }
+
+  static initFromFile(filename: string, peerId: PeerId): FileENR {
```

Comment:
We will likely want/need another static method `initFromENR(filename: string, peerId: PeerId, enr: ENR): FileENR`
to be used :point_down: in `src/network/enr.ts`